### PR TITLE
docs(readme): Apdex integration

### PR DESCRIPTION
**Pull request by vmakhaev**
**Summary**
> What kind of change does this PR introduce?

Add integration for: [Apdex](https://apdex.co)

Apdex uses Lighthouse for website performance monitoring.
Thank you!

### CODE REVIEW

**Comment by vmakhaev** [comment]
@googlebot I signed it!

**Comment by vmakhaev** [comment]
@adamraine it's just documentation change. I tried to run two time, but different tests are failing each time. Should I keep restarting?

**Comment by patrickhulce** [comment]
@vmakhaev it looks like CI is failing to download the build dependencies. Let's give it a break for a bit but then yes, just restarting should fix.

File: readme.md [hunk 1]
```diff
@@ -339,6 +339,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Foo](https://www.foo.software/lighthouse)** - Lighthouse-as-a-service offering free and premium plans. Provides monitoring and historical reporting of Lighthouse audits with CircleCI, GitHub, and other integrations. Features include Slack notifications, PR comment reporting and more.
 
+* **[Apdex](https://apdex.co/performance)** - Apdex is website performance service. Provides historical Lighthouse reports visializations, global regions, mobile/desktop option, custom headers and alerts. It's also all-in-one solution, providing website uptime monitoring and more. Apdex is paid product with free 30-day trial and flexible plans.
```
Comment (patrickhulce) [inline readme.md:342]:
```suggestion
* **[Apdex](https://apdex.co/performance)** - Apdex is a website performance monitoring service. Provides historical Lighthouse report visualizations, global region options, mobile+desktop support, custom headers, alerts, and uptime monitoring. Apdex is paid product with free 30-day trial and flexible plans.
```

just some consolidation and minor grammar fixes for you to consider :)

**Review by patrickhulce — COMMENTED** [review]
neat @vmakhaev thanks for sharing!

**Review by adamraine — APPROVED** [review]
LGTM pending @patrickhulce's  suggestions

**Comment by vmakhaev** [comment]
I just made the text a little bit smoother

File: readme.md [hunk 2]
```diff
@@ -339,6 +339,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Foo](https://www.foo.software/lighthouse)** - Lighthouse-as-a-service offering free and premium plans. Provides monitoring and historical reporting of Lighthouse audits with CircleCI, GitHub, and other integrations. Features include Slack notifications, PR comment reporting and more.
 
+* **[Apdex](https://apdex.co)** - Apdex is a website performance service. The main features are historical Lighthouse reports visualizations, mobile/desktop options, alerts, uptime monitoring, and more. There are flexible plans and a 30-day free trial.
```
Comment (patrickhulce) [inline readme.md:342]:
```suggestion
* **[Apdex](https://apdex.co)** - Apdex is a website performance service. The main features are historical Lighthouse report visualizations, mobile/desktop options, alerts, uptime monitoring, and more. There are flexible paid plans and a 30-day free trial.
```

**Review by patrickhulce — APPROVED** [review]
thanks @vmakhaev!

**Pull request merged**